assets/sparksql/Dockerfile

```
@@ -0,0 +1,13 @@
+# FROM spark:python3
+
+FROM spark:4.0.0
+
+USER root
+
+RUN apt-get update;
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)